src/mono/wasm/debugger/DebuggerTestSuite/DebuggerTestBase.cs

```diff
@@ -37,6 +37,18 @@ public static WasmHost RunningOn
 #else
             => WasmHost.Firefox;
 #endif
+
+        public static bool WasmEnableThreads 
+        {
+            get
+            {
+                var envVar = Environment.GetEnvironmentVariable("WASM_TESTS_USING_VARIANT");
+                if (envVar == "multithreaded")
+                    return true;
+                return false;
+            }
+        }
```

Comment:
Instead of this, add:
```csharp
public static readonly string? WasmTestsUsingVariant = Environment.GetEnvironmentVariable("WASM_TESTS_USING_VARIANT");
```
.. to `EnvironmentVariables.cs`.

```suggestion
        public static bool WasmEnableThreads => EnvironmentVariables.WasmTestsUsingVariant == "multithreaded";
```

Reply:
done.